### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alberttwong**
added clarification on the config file and how to submit it into kafka.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The connector supports:
 - batching;
 - delivery retries.
 
-An example of configuration:
+An example of configuration (file connect-http-sink.json):
```
Comment (jeqo) [inline README.md:21]:
```suggestion
An example of configuration (file `connect-http-sink.json`):
```

File: README.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ An example of configuration:
 }
 ```
 
+Then execute the config json against the Apache Kafka REST API `curl -d @config/connect-http-sink.json -H "Content-Type: application/json" -X POST http://localhost:8083/connector`
```
Comment (jeqo) [inline README.md:40]:
```suggestion
Then deploy the connector configuration against the Kafka Connect REST API with the following command: 

```shell
curl -d @config/connect-http-sink.json -H "Content-Type: application/json" -X POST http://localhost:8083/connector
```

```

**Review by jeqo — CHANGES_REQUESTED** [review]
@alberttwong thanks! Just a couple of minor improvements.

**Comment by alberttwong** [comment]
done.

File: README.md [hunk 3]
```diff
@@ -37,6 +37,12 @@ An example of configuration:
 }
 ```
 
+Then deploy the connector configuration against the Kafka Connect REST API with the following command:
```
Comment (jeqo) [inline README.md:40]:
```suggestion
Then deploy the connector configuration against the Kafka Connect REST API with the following command:
```

**Review by jeqo — CHANGES_REQUESTED** [review]
Thanks @alberttwong ! There's a last failing bit.

You can validate build process with `./gradlew :spotlessMiscCheck`

**Review by jeqo — APPROVED** [review]
Thanks @alberttwong !

**Pull request merged**